### DISCUSSION TITLE
Desktop zoom

### DISCUSSION THREAD

**Discussion by edmundlaugasson** [elementary/triage#397] [Q&A] [answered]
Would propose to add built-in whole desktop zoom regardless of opened apps via GUI configure option (which key to use, default zoom factor, quick reset key to restore unzoomed view) and turned on by default e.g. via ALT+mouse scroll like under Linux Mint XFCE. [One successful attempt found at 2012](https://heathpaddock.com/?p=523). As switching to Wayland, this article might not work in case of 8.x version. E.g. in Linux Mint XFCE there is already such whole desktop zoom feature working right away after installation and not only - even during installation.
Desktop zoom is very handy feature, especially in case of special needs - impaired vision. Also when eyes are tired, we need enlarge desktop. Also in case of teaching we need emphasize certain part of desktop and make it more visible via multimedia projector - we need to enlarge whole desktop and follow mouse cursor when zoomed in.

**Comment by lenemter** [comment, marked as answer]
There is zoom in elementary OS. You can access it using `Super`+`+/-` shortcuts.

> via ALT+mouse scroll

This was reported in https://github.com/elementary/gala/issues/815

> even during installation

https://github.com/elementary/greeter/issues/540

**Reply by danirabbit** [reply]
We also support multitouch gestures here which can be set up in System Settings → Mouse & Touchpad → Gestures
